Tools/Scripts/webkitpy/layout_tests/controllers/layout_test_runner.py

```diff
@@ -570,6 +570,15 @@ def __eq__(self, other):
 
 
 class Sharder(object):
+    MAX_SIZE = 256
+
+    @classmethod
+    def bucket_for(cls, string, limit=2):
+        result = 0
+        for c in string:
+            result = (result + ord(c)) % limit
+        return result
```

Comment:
is there any reason not to go for something like `hashlib.sha1` here, rather than trying to define our own hash function (which probably isn't uniform at all, therefore won't result in roughly equal chunk sizes)?